testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -51,6 +51,45 @@ def test_sct_label_vertebrae_high_value_warning(caplog, tmp_path):
     assert 'Disc value not included in template.' in caplog.text
 
 
+def test_sct_label_vertebrae_initial_disc_no_inferior(caplog, tmp_path):
+    caplog.set_level(logging.INFO)
+    sct_label_vertebrae.main(['-i', sct_test_path('t2', 't2.nii.gz'),
+                              '-s', sct_test_path('t2', 't2_seg-manual.nii.gz'),
+                              '-c', 't2', '-initz', '40,20',
+                              '-ofolder', str(tmp_path)])
+    assert 'No disc is inferior to the initial disc.' in caplog.text
+
+
+def test_sct_label_vertebrae_initial_disc_zero(capsys, tmp_path):
+    with pytest.raises(SystemExit) as excinfo:
+        sct_label_vertebrae.main(['-i', sct_test_path('t2', 't2.nii.gz'),
+                                  '-s', sct_test_path('t2', 't2_seg-manual.nii.gz'),
+                                  '-c', 't2', '-initz', '40,0',
+                                  '-ofolder', str(tmp_path)])
+    assert excinfo.value.code == 1
+    assert 'Missing label or zero label for initial disc.' in capsys.readouterr().out
+
+
+def test_sct_label_vertebrae_initial_disc_too_low(capsys, tmp_path):
```

Comment:
Actually, wait, sorry. It occurs to me that none of the error messages observed from the tests (https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3775#pullrequestreview-950071054) match the error message in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3775#issue-1210242621.

Did we want to try and create a test that (would have) thrown the `UnboundLocalError`?